### ISSUE TITLE
Double click on a word sometimes selects the whole paragraph

### ISSUE THREAD

**Issue by RWeigelt** [typora/typora-issues#4246]
Version 0.9.98 (beta), Windows 10, Usage in default mode (i.e. not typewriter, not focus, not source code).

When I double-click a word inside a paragraph to select the word, sometimes the whole paragraph is selected.
There seems to be an issue with the detection of a double vs. a triple click.

For comparison, I just tried double click vs. triple click here on the GitHub website, in the text field I am typing this issue into. The difference is detected reliably so I never accidently select the whole paragraph if I do not intend to do it.

**Comment by abnerlee** [comment]
> For comparison, I just tried double click vs. triple click here on the GitHub website, in the text field I am typing this issue into. 

Did you try with chrome?

---

BTW, you mean single click + double click or double click + single click be treated as triple click, right? If not, could you attach a screencast?

**Comment by RWeigelt** [comment]
After trying around some more, it looks more and more like a "user error" on my side. The triple click occurs when I click into a word and then decide to select it by double clicking too soon. Yesterday, when I wrote the issue, I could not exactly pin-point the cause.

When I compared Typora and the Github input field, I had the browser window at a different position on the monitor than the Typora window. This means different hand positions for the mouse which could have influenced the timing. Today I could not notice a real difference between the two.

I checked my mouse settings. On my system, I had a double click speed of 410 (from ``HKEY_CURRENT_USER\Control Panel\Mouse``, value of ``DoubleClickSpeed``, this is one notch right from the center in the Mouse Properties ). Now that I have changed the value to 340 (another notch to the right), I have a hard time reproducing the accidental triple-click.

I'd say, close this issue.

BTW: Awesome software!
